github-authn-jwt.yml

```diff
@@ -0,0 +1,56 @@
+- !policy
+  id: conjur/authn-jwt/github
+  body:
+      - !webservice
+
+      #Mandatory variable: The JWT provider URI
+      #Uncomment either 'provider-uri' OR 'jwks-uri'
+
+      #- !variable
+      #    id: provider-uri
+
+      - !variable
+        id: jwks-uri
```

Comment:
Given my past experience with solving internal & customer complications with understanding the JWT Authenticator configuration, could you please add a comment on the end of the line of an example value that would be set to the variable?

Reply:
Sure. Updated.